src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java

```diff
@@ -119,8 +119,11 @@ public void verifyThatSurveillanceDataIsSavedInReport(final String friendlySurvI
         /* Workaround to resolve Stale Element Reference Exception-
          * reload the element after form update so it's attached to DOM: https://www.seleniumhq.org/exceptions/stale_element_reference.jsp*/
         getDriver().get(getUrl() + "#/surveillance/reporting");
+        getDriver().navigate().refresh();
         steps.iExpandACB("UL LLC");
-        SurveillanceReportPage.editSurveillanceReport(getDriver(), "UL LLC-2019-Q1").click();
+        WebElement editbutton = SurveillanceReportPage.editSurveillanceReport(getDriver(), "UL LLC-2019-Q1");
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].scrollIntoView(true);", editbutton);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", editbutton);
```

Comment:
After clicking this button, there's a popup about "are you sure you want to initiate surveillance", so the next few lines can't happen

Reply:
This is for edit button, not initiate

Reply:
It is for edit button and not initiate